qa-pipelines/tasks/cf-upgrade.sh

```diff
@@ -80,25 +85,15 @@ if [[ "${EMBEDDED_UAA:-false}" != "true" ]]; then
 fi
 
 # Deploy CF
-set_helm_params # Resets HELM_PARAMS
-set_scf_sizing_params # Adds scf sizing params to HELM_PARAMS
+set_helm_params # Resets HELM_PARAMS.
+set_scf_params # Adds scf specific params to HELM_PARAMS.
 
+# Explicitly setting mysql count to 1 for pxc upgrade testing for scf.
 if pxc_post_upgrade; then
-  #HELM_PARAMS+=(--set=config.HA_strict=false)
   HELM_PARAMS+=(--set=sizing.mysql.count=1)
 fi
 
-if [[ "${EMBEDDED_UAA:-false}" != "true" ]]; then
```

Comment:
Move these back for right now because we don't have a good way of checking the task type from here right now (need refactoring)